packages/shared/src/components/MobileHeaderRankProgress.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { CSSProperties, ReactElement } from 'react';
+import useReadingRank from '../hooks/useReadingRank';
+import RankProgressWrapper from './RankProgressWrapper';
+import { rankToColor } from '../lib/rank';
+
+export default function MobileHeaderRankProgress(): ReactElement {
+  const { isLoading, rank, nextRank } = useReadingRank();
+
+  if (isLoading) {
+    return <div className="">&nbsp;</div>;
+  }
+
+  return (
+    <div
```

Comment:
Why do we need this parent `div`?

Reply:
We actually need it to use the top header space-between.
Else the button itself will just span across all available space.